src/Wallabag/CoreBundle/Resources/config/services.yml

```diff
@@ -44,11 +44,27 @@ services:
         class: Graby\Graby
         arguments:
             - { error_message: false }
+            - @wallabag_core.http_client
         calls:
             - [ setLogger, [ @logger ] ]
         tags:
             - { name: monolog.logger, channel: graby }
 
+    wallabag_core.http_client:
+        class: GuzzleHttp\Client
+        arguments:
+            - {"defaults": {"cookies": @wallabag_core.authenticator.cookie_jar}}
```

Comment:
Do not forget to re-use SafeCurlHandler here (which increase protection of cURL).
Something like:

```json
    wallabag_core.http_client:
        class: GuzzleHttp\Client
        arguments:
            -
                defaults: {"cookies": "@wallabag_core.authenticator.cookie_jar"}
                handler: "@wallabag_core.graby.safecurl_handler"

    wallabag_core.graby.safecurl_handler:
        class: Graby\Ring\Client\SafeCurlHandler
```